52.mender-gateway/docs.md

```diff
@@ -8,6 +8,22 @@ github: false
 ---
 
 
+## mender-gateway 1.0.1
+
+_Released 09.25.2022_
+
+### Changelogs
+
+#### mender-gateway (1.0.1)
+
+New changes in mender-gateway since 1.0.0:
+
+##### Features
+
+* Licenses are now available in the package, instead of only
```

Comment:
Related to above: Here we list it as a feature (also), but we do a patch bump?

Reply:
Same as above